server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -79,10 +81,12 @@ export class TenantManager {
             if (isNetworkError(error)) {
                 if (error.code === 403 && !tenantKeys.key2){
                     // Delete key from cache when there is a 403 error and no key2 to validate
-                    Lumberjack.info(`Error with tenant key 1 token verification while key 2 is not present. Deleting key from cache.`, lumberProperties);
-                    await this.deleteKeyFromCache(tenantId).catch((err) => {
-                        Lumberjack.error(`Error deleting keys from the cache.`, lumberProperties, err);
-                    });
+                    if (bypassCache && this.isCacheEnabled) {
```

Comment:
I might be mistaken, but if `bypassCache` is `true`, then we would not want to do operations against the cache. Is that correct?

If yes, should this be
`if (!bypassCache && this.isCacheEnabled) {`
?

Reply:
I replaced this variable using a built-in tool in my IDE, and must've missed the "!" Thanks for catching this.